.github/workflows/ci.yml

```diff
@@ -27,22 +27,32 @@ jobs:
           - "24.3"
```

Comment:
Would you mind also bumping OTP 26 to 26.3?

Reply:
Not at all 😄

Reply:
9fe1749

Reply:
Ouch, it seems the most recent OTP-26 is 26.2, there were no 26.3 yet, sorry...

Reply:
Yeah, I could've checked too 😄 No worries, I'll force push to fix the latest commit.